### PR TITLE
Feature/website

### PR DESCRIPTION
**Pull request by gitboej**
This is the first draft of my personal website

### CODE REVIEW

**Comment by peterpane1990** [comment]
looks dope!! and as far as i can see, no mistakes! 👌

a href="mailto:kontakt@email.de"> Ankertext (i had to leave out the opening and closing brackets, otherwise it would not show the code)
Maybe you want to add this? That way, by clicking on your mail, it will open a new mail with the address already filled out.

**Review by peterpane1990 — APPROVED** [review]
oops, forgot to approve ;)

**Review by peterpane1990 — APPROVED** [review]
nice!

File: mywebsite/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      <nav>
+        <a href="">Contact</a>
+        <a href="">Biography</a>
+      </nav>
+    </header>
+    <section id="picture">
+      <article>
+        <img
+          src="assets/JPEG_20200526_043724_3762250490343282501.png"
+          alt="Picture of Lars Bonus"
```
Comment (t0bbl) [inline mywebsite/index.html:21]: nice bonus!

**Pull request merged**